src/OrchardCoreContrib.PoExtractor/PluginHelper.cs

```diff
@@ -0,0 +1,55 @@
+using System.Reflection;
+using Microsoft.CodeAnalysis.CSharp.Scripting;
+using Microsoft.CodeAnalysis.Scripting;
+using OrchardCoreContrib.PoExtractor.DotNet;
+using OrchardCoreContrib.PoExtractor.DotNet.CS;
+using OrchardCoreContrib.PoExtractor.DotNet.VB;
+using OrchardCoreContrib.PoExtractor.Liquid;
+using OrchardCoreContrib.PoExtractor.Razor;
+
+namespace OrchardCoreContrib.PoExtractor;
+
+public static class PluginHelper
+{
+    public static async Task ProcessPluginsAsync(
+        IEnumerable<string> plugins,
+        List<IProjectProcessor> projectProcessors,
+        List<string> projectFiles,
+        IEnumerable<Assembly> assemblies = null)
+    {
+        assemblies ??=
+        [
+            typeof(IProjectProcessor).Assembly, // OrchardCoreContrib.PoExtractor.Abstractions
+            typeof(ExtractingCodeWalker).Assembly, // OrchardCoreContrib.PoExtractor.DotNet
```

Comment:
Why this is needed?!!

Reply:
I wanted to add access to the full assembly, in case the plugin writer wants to derive a type from something existing. Like the way `RazorProjectProcessor` inherits from `CSharpProjectProcessor`.

But I just checked and [only adding `Program`'s assembly](https://github.com/OrchardCoreContrib/OrchardCoreContrib.PoExtractor/pull/102/commits/04f66300831ec28999bedfb3be4f529a6e9b1631) is enough. So I've updated the code accordingly.